automation/run_python_tests.sh

```diff
@@ -2,16 +2,14 @@
 
 set -e
 
-cd components/nimbus
-cargo build --no-default-features
-cargo uniffi-bindgen generate src/cirrus.udl --language python -o .
-cargo build --manifest-path ../../megazords/cirrus/Cargo.toml --release
+cargo uniffi-bindgen generate components/nimbus/src/cirrus.udl --language python -o .
+cargo uniffi-bindgen generate components/support/nimbus-fml/src/fml.udl --language python -o .
+cargo build --manifest-path megazords/cirrus/Cargo.toml --release
 
 if [[ "$OSTYPE" == "darwin"* ]]; then
-  mv ../../target/release/libcirrus.dylib ./libuniffi_cirrus.dylib
+  mv target/release/libcirrus.dylib ./
 else
-  mv ../../target/release/libcirrus.so ./libuniffi_cirrus.so
+  mv target/release/libcirrus.so ./
 fi
```

Comment:
@yashikakhurana This PR will change the filename of the binary it generates for the Cirrus megazord — you'll need to make a similar adjustment in the Cirrus Dockerfile (sorry). 

I could probably force it to be the same filename but I'd prefer not to because setting the `cdylib_name` to `uniffi_cirrus` feels odd to me. Not sure why, I could probably be convinced to switch to that.